locale/fi/LC_MESSAGES/django.po

```diff
@@ -3688,3 +3698,1254 @@ msgstr "Unohda automaattinen uudelleenohjaus."
 #: userprofile/views.py:80
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
+#: venv/lib/python3.7/site-packages/colorfield/fields.py:10
```

Comment:
Please create messages with `./manage.py createmessages --ignore='venv/*' --no-obsolete`.

Reply:
I guess you mean `makemessages`? Or am I missing an undocumented command?

Additionally, can this be added to the development documentation somewhere?

Reply:
> I guess you mean makemessages?

Yes, my bad.

p.s. how did I miss this response. I checked all the conversaionts from the notification. Sorry for that.

> Additionally, can this be added to the development documentation somewhere?

Yes. I think it should be. Specially that --no-obsolete. Ignore is only required if the virtual env is installed within the repository. I might add this to the styleguide (i.e. how translations should be handled with the git).